components/counter/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "@auro-formkit/auro-counter",
+  "version": "1.0.0",
+  "description": "auro-counter HTML custom element",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/AlaskaAirlines/auro-counter"
+  },
+  "type": "module",
+  "main": "dist/index.js",
+  "module": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "license": "Apache-2.0",
+  "dependencies": {
+    "chalk": "^5.2.0",
+    "lit": "^3.2.1"
+  },
+  "peerDependencies": {
+    "@aurodesignsystem/design-tokens": "^4.9.2",
+    "@aurodesignsystem/webcorestylesheets": "^5.1.2"
+  },
+  "devDependencies": {
+    "@auro-formkit/build-tools": "*",
+    "@auro-formkit/config": "*",
+    "@auro-formkit/typescript": "*",
+    "@aurodesignsystem/design-tokens": "^4.9.2",
+    "@aurodesignsystem/webcorestylesheets": "^5.1.2",
+    "@rollup/plugin-node-resolve": "^15.3.0",
+    "postcss-selector-replace": "^1.0.2",
+    "rollup": "^4.22.4",
+    "rollup-plugin-serve": "^2.0.2",
+    "wc-sass-render": "^1.4.0",
+    "web-component-analyzer": "^2.0.0"
+  },
+  "scripts": {
+    "build": "run-s build:sass bundler types",
+    "build:sass": "sass --no-source-map --load-path=../../node_modules \"./src/styles/:./src/styles/\"",
+    "build:watch": "nodemon -e scss,js --watch src --ignore \"*.css\" --ignore \"*-css.js\" --exec \"npm run build\"",
+    "bundler": "rollup -c node:@auro-formkit/config/rollup --input src/index.js",
```

Comment:
@sun-mota [landed a change earlier](https://github.com/AlaskaAirlines/auro-form/pull/88) that removes the `--input`flag:

It should now be:

```
"bundler": "rollup -c node:@auro-formkit/config/rollup",
```

Reply:
Updated